.github/workflows/publish.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   publish:
     name: Publish on NuGet
     runs-on: ubuntu-latest
-    if: ${{ github.repository_owner == env.REPOSITORY_OWNER }}
+    if: ${{ github.repository_owner == vars.REPOSITORY_OWNER }}
```

Comment:
According to https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables, the variable name should be GITHUB_REPOSITORY_OWNER:

![image](https://github.com/marcominerva/OperationResults/assets/3522534/db5cd578-923d-44fd-8ebf-a09aaf88f32a)

Moreover, are you sure that this information is accessible also trough `vars`?